docker/production/Dockerfile

```diff
@@ -15,33 +15,34 @@ EXPOSE 3000
 
 ENTRYPOINT ["/usr/src/app/docker/entrypoint-worker.sh"]
 
-# https://github.com/nodesource/distributions#installation-instructions
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
-
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-
-RUN apt-get update && apt-get install -y nodejs yarn ffmpeg imagemagick pdftk ghostscript sqlite3 rsync shared-mime-info --no-install-recommends && rm -rf /var/lib/apt/lists/*
-
-
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash \
- && apt-get update && apt-get install -y nodejs && rm -rf /var/lib/apt/lists/* \
- && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
- && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
- && apt-get update && apt-get install -y yarn && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ffmpeg imagemagick pdftk ghostscript sqlite3 rsync shared-mime-info && \
+    rm -rf /var/lib/apt/lists/* && \
+    curl -sL https://deb.nodesource.com/setup_14.x | bash && \
+    apt-get update && \
+    apt-get install -y nodejs && \
+    rm -rf /var/lib/apt/lists/* && \
```

Comment:
This is also redundant

Reply:
How is it redundant? 
According to [the script](https://deb.nodesource.com/setup_14.x) itself it just installs the repo and not the program:
```
#!/bin/bash

# Discussion, issues and change requests at:
#   https://github.com/nodesource/distributions
#
# Script to install the NodeSource Node.js 14.x repo onto a
# Debian or Ubuntu system.
#
# Run as root or insert `sudo -E` before `bash`:
#
# curl -sL https://deb.nodesource.com/setup_14.x | bash -
#   or
# wget -qO- https://deb.nodesource.com/setup_14.x | bash -
#
# CONTRIBUTIONS TO THIS SCRIPT
#
# This script is built from a template in
# https://github.com/nodesource/distributions/tree/master/deb/src
# please don't submit pull requests against the built scripts.
```

Reply:
Resolving because of missing feedback!